doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -1,50 +1,52 @@
 """
-================================================================
+==============================================================
 Use rolling-ball algorithm for estimating background intensity
-================================================================
+==============================================================
 
 The rolling-ball algorithm estimates the background intensity of a grayscale
-image in case of uneven exposure. It is frequently used in biomedical
+image. It comes in useful, for instance, in case of uneven exposure, when
+subtracting the background is desirable. It is frequently used in biomedical
 image processing and was first proposed by Stanley R. Sternberg in
 1983 [1]_.
 
-The algorithm works as a filter and is quite intuitive. We think of the image
+The algorithm works as a filter: think of the image
```

Comment:
```suggestion
The algorithm works as a filter: Think of the image
```
Just a (possible, not strict) US spelling convention, since what follows the colon is an independent clause.

Reference: https://www.grammarly.com/blog/punctuation-capitalization/capitalization-after-colons/